scripts/scil_bundle_diameter.py

```diff
@@ -78,6 +84,9 @@ def _build_arg_parser():
     p2.add_argument('--opacity', type=float, default=0.2,
                     help='Opacity for the streamlines rendered with the tube.'
                     '\n[Default: %(default)s]')
+    p2.add_argument("--win_dims", nargs=2, metavar=("WIDTH", "HEIGHT"),
+                    default=(1920, 1080), type=int,
```

Comment:
I would remove the parenthesis in the metavar, so that users don't think they need to put it.

Reply:
Do you mean in the default value displayed ? The formatting of the help message is not right, it should look like [Default : (1920, 1080)], or [Default : [1920, 1080]] if I use a list to specify it.